### PR TITLE
Second post named as WebsiteInfo.md created

### PR DESCRIPTION
**Pull request by Renganathan-R**
In this new branch, I added a markdown file named WebsiteInfo that tells more about my new site.

### CODE REVIEW

File: WebsiteInfo.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Hi users!
```
Comment (amjith) [inline WebsiteInfo.md:1]: This should not be a title.

File: WebsiteInfo.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Once again welcome all!
```
Comment (amjith) [inline WebsiteInfo.md:3]: Don't add sentences like welcome again. It is presumptuous to assume that you have repeated readers of your blog. Just write blog as if you're the only audience who will be reading it.

Comment (Renganathan-R) [reply WebsiteInfo.md]: I changed the contents as per your suggestions

**Comment by amjith** [comment]
Excellent. Looks great! Merge it in. :1st_place_medal:

**Pull request merged**